lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -152,6 +152,7 @@ pyuwsgi==2.0.18.post0
 pyyaml==5.3
 rdflib-jsonld==0.4.0
 rdflib==4.2.2
+refgenconf==0.7.0
```

Comment:
Can you please add `refgenconf` to  lib/galaxy/dependencies/pipfiles/default/Pipfile

Reply:
Yes, nice catch. Thanks.